routing_optimization_over_server/cvrptw_service_team_routing.ipynb

```diff
@@ -0,0 +1,413 @@
+{
```

Comment:
Same link over in this notebook as well, lets update that as well.

Reply:
removed